static/js/modules/columns.js

```diff
@@ -68,34 +236,111 @@ var filings = {
+  total_receipts: currencyColumn({data: 'total_receipts', className: 'min-tablet hide-panel'}),
+  total_disbursements: currencyColumn({data: 'total_disbursements', className: 'min-tablet hide-panel'}),
+  total_independent_expenditures: currencyColumn({data: 'total_independent_expenditures', className: 'min-tablet hide-panel'}),
+  modal_trigger: modalTriggerColumn
+};
+
+var independentExpenditures = [
+  {
+    data: 'committee',
     orderable: false,
+    className: 'all',
     render: function(data, type, row, meta) {
-      return row.form_type && ['F3', 'F3P', 'F3X'].indexOf(row.form_type) !== -1 ?
-        tables.MODAL_TRIGGER_HTML :
-        '';
+      if (data) {
+        return columnHelpers.buildEntityLink(
+          data.name,
+          helpers.buildAppUrl(['committee', data.committee_id]),
+          'committee'
+        );
+      } else {
+        return '';
+      }
     }
-  }
-};
+  },
+  currencyColumn({data: 'expenditure_amount', className: 'min-tablet'}),
+  {
+    data: 'candidate_name',
+    orderable: false,
+    className: 'min-tablet hide-panel',
+    render: function(data, type, row, meta) {
+      if (row.candidate_id) {
+        return columnHelpers.buildEntityLink(
+          data,
+          helpers.buildAppUrl(['candidate', row.candidate_id], tables.getCycle(row, meta)),
+          'candidate'
+        );
+      } else {
+        return row.candidate_name;
+      }
+    }
+  },
+  _.extend({}, supportOpposeColumn, {className: 'min-tablet'}),
+  dateColumn({data: 'expenditure_date', className: 'min-desktop hide-panel-tablet'}),
+  columnHelpers.urlColumn('pdf_url', {data: 'expenditure_description', className: 'min-desktop hide-panel', orderable: false}),
+  modalTriggerColumn
+];
+
+var receipts = [
+  {
+    data: 'contributor',
+    orderable: false,
+    className: 'all',
+    width: '200px',
+    render: function(data, type, row, meta) {
+      if (data && row.receipt_type !== helpers.globals.EARMARKED_CODE) {
+        return columnHelpers.buildEntityLink(
+          data.name,
+          helpers.buildAppUrl(['committee', data.committee_id]),
+          'committee'
+        );
+      } else {
+        return row.contributor_name;
+      }
+    }
+  },
+  {data: 'contributor_state', orderable: false, className: 'min-desktop hide-panel'},
+  {data: 'contributor_employer', orderable: false, className: 'min-desktop hide-panel'},
+  currencyColumn({data: 'contribution_receipt_amount', className: 'min-tablet'}),
+  dateColumn({data: 'contribution_receipt_date', className: 'min-tablet hide-panel-tablet'}),
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'min-desktop hide-panel',
+    render: function(data, type, row, meta) {
+      if (data) {
+        return columnHelpers.buildEntityLink(
+          data.name,
+          helpers.buildAppUrl(['committee', data.committee_id]),
+          'committee'
+        );
+      } else {
+        return '';
+      }
+    }
+  },
+  modalTriggerColumn
+];
```

Comment:
Yep, I agree. I started to do this, but then realized that many of the `committee` and `candidate` columns vary just _slightly_ across the different pages, due to different ways the API returns results.

I still think it's worthwhile, but decided to keep these as-is in the interest of time (and not accidentally breaking something).